### DISCUSSION TITLE
Need help! how do i make my Reference line looks exactly like this  and shows the current time

### DISCUSSION THREAD

**Discussion by ArthurEmma2** [recharts/recharts#4391] [Q&A] [answered]
![current](https://github.com/recharts/recharts/assets/86601468/de73e47a-e364-4c5b-b8cc-09d973fdad74)

**Comment by ckifer** [comment]
Can you make a codesandbox representing what you have/what you have tried and with some example data?

**Reply by ckifer** [reply, marked as answer]
[![Edit reference-line-custom-shape-label](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/reference-line-ifoverflow-extenddomain-forked-65xc98)

This is a start, but the arrow will have to be something you add custom. You can see to add the dot I just added a `circle` in the custom shape I made. You'll have to do the same at x2, y2 but use an arrow shape

**Reply by ckifer** [reply]
![image](https://github.com/recharts/recharts/assets/25180830/504e6029-560e-4683-a69a-db7081b7b554)

**Reply by ckifer** [reply]
![image](https://github.com/recharts/recharts/assets/25180830/732afe9a-a3ff-4eb9-8b89-6fcbb0107188)
added arrow but customizing it is up to you
